recipes/openfaas.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: kbrew
+app:
+  args:
+    functionNamespace: openfaas-fn
+    generateBasicAuth: true
+  repository:
+    name: openfaas
+    url: https://openfaas.github.io/faas-netes/
+    type: helm
+  namespace: "openfaas"
+  sha256:
+  version: 8.0.2
+  pre_install:
+  - steps:
+    - kubectl apply -f https://raw.githubusercontent.com/openfaas/faas-netes/master/namespaces.yml
+  post_cleanup:
+    steps:
+    - kubectl delete ns openfaas-fn
```

Comment:
I think we should delete `openfaas` namespace too?

Reply:
hi @PrasadG193 ,  i thought it gets delete autmaticaly, wenn the chart is gone! Ok np!